src/main/java/com/radixdlt/crypto/Signatures.java

```diff
@@ -0,0 +1,66 @@
+package com.radixdlt.crypto;
+
+import java.util.Map;
+
+public interface Signatures {
```

Comment:
Not clear to me what `Signatures` represents. Is it a homogenous collection of signatures? If so, why is it not generic? Or is it just for EC signatures? Is it a utility class, a collection, ..?

Reply:
Me and @clstrfsck tried to make it Generic, but we failed, hmm, can't remember why now. In Swift it would have an `associattype SignatureType: Signature`

Reply:
Or nah, right I think I did not want to make it generic, so that we did not have to surface the type to e.g. `QuorumCertificate`, where it's used.